### ISSUE TITLE
Scrubbing

### ISSUE THREAD

**Issue by woolmonkey** [jellyfin/jellyfin-androidtv#3585]
### Describe the feature you'd like

So now that scrubbing is a built in Jellyfin feature and not a plugin, any plan to support it?

**Comment by nielsvanvelzen** [comment]
Do you mean the TrickPlay functionality?

**Comment by woolmonkey** [comment]
Yes.  When moving on timeline to get a view of the video instead of nothing.  I know it was asked before with Jellyscrub and the answer was always only support official features, so just wondering now that's official if there are plans for it.

**Comment by danni-storm** [comment]
This is a duplicate of closed issues #2504 and #3316. However, as @woolmonkey mentioned, TrickPlay has been merged with base Jellyfin, so the reasoning behind the closure of these issues has been addressed. 

(mostly here to document the relevant past issues so no one has to go digging for them)

**Comment by woolmonkey** [comment]
Thanks for the update I was just wondering.

**Comment by brianjmurrell** [comment]
> TrickPlay is planned for the new video player (details in #1057),

TrickPlay doesn't seem to be mentioned in that issue at all.  Is that just oversight or is there intentionally no "TODO" item for it in that issue?  Just trying to figure out which issues to subscribe to stay updated on this particular feature.

Thanks for all of your work on this app.  It's really the heart and soul of what makes Jellyfin useful (to me, and I am sure lots of others, at least).
